crt_portal/static/sass/custom/form.scss

```diff
@@ -7,11 +7,6 @@ form#report-form {
   .question-header {
     margin-top: 2.5rem;
     margin-bottom: 1.5rem;
-    + .help-text,
-    + .help-text__small,
-    + p {
-      margin-top: -1.5rem; // to match other help text margins
-    }
```

Comment:
Still working on this part -- not entirely comfortable removing it just yet (mostly because I'm not 100% sure what the intended effect was). This bit makes the help text paragraph closer to its header for some sections (e.g., the hate crimes and human trafficking section) but its making help text in other sections inconsistent. Not sure what to do here, open to ideas...

Reply:
To my knowledge, we have three places where this CSS applies: hate crimes, location (edu) and personal characteristics. Removing this whole bit seems to fix the issues we are seeing currently but I may be missing prior history + knowledge of why this change was applied.

Reply:
so help text is supposed to be 8px away from whatever it follows, but `.question-header` isn't always followed by helptext. When it's followed by anything else, the spacing is supposed to be 24px instead.

For instance, the blue text here has the `question-header` class:
<img width="497" alt="Screen Shot 2020-05-12 at 3 25 22 PM" src="https://user-images.githubusercontent.com/509309/81747190-dfc39a00-9464-11ea-9006-8a3034f86d54.png">

I did a little digging, and it looks like both the hate crimes and the personal characteristics help text `p` tags are missing the `help-text` class altogether. Instead, the class is showing up on `em` tags inside the `p` tags, and the `p` tags have a bunch of extraneous margin classes on them:
<img width="535" alt="Screen Shot 2020-05-12 at 3 28 19 PM" src="https://user-images.githubusercontent.com/509309/81747383-4052d700-9465-11ea-9e31-f9c8f328a436.png">

We should probably just rewrite the help text markup in these couple locations so they match the help text markup on the other pages. 😅

Reply:
I just did some local fiddling, and between fixing the markup and changing this:
```+ .help-text,
    + .help-text__small,
    p {
      margin-top: -1.5rem; // to match other help text margins
    }```

to this:
```+ .help-text,
    + .help-text__small,
    p {
      margin-top: -1rem; // to match other help text margins
    }```

everything seems to work as expected!

Reply:
Ooooh that makes more sense now. Thanks! (BTW: feel free to push up to this branch, will add Co-authored-by trailers)

Reply:
Apropos of nothing, seeing the `None` in that class makes me sad.